projects/cm_mcu/LocalTasks.c

```diff
@@ -195,6 +200,21 @@ struct sm_command_t sm_command_fflit_f1[] = {
     {1, 0x00, 0x16, 2, "FF_TEMPERATURE", 0xff, "C", PM_STATUS},
     {2, 0x00, 0x07, 1, "FF_LOS_ALARM", 0xffff, "", PM_STATUS},
     {2, 0x00, 0x14, 1, "FF_CDR_LOL_ALARM", 0xffff, "", PM_STATUS},
+    // there are no registers to read optical power for 14Gbps ECUO.
+    // registers below are a placeholder with a reading equal to zero
+    // the reason we need them because n_commands is fixed
+    {1, 0x00, 0x00, 1, "FF_CH01_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH02_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH03_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH04_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH05_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH06_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH07_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH08_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH09_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH10_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH11_OPT_POW", 0xff, "mw", PM_STATUS},
+    {1, 0x00, 0x00, 1, "FF_CH12_OPT_POW", 0xff, "mw", PM_STATUS},
```

Comment:
i understand why you did it this way, but we should think of a better solution than this to solve this problem

Reply:
the issue has something to do with the size of commands and outputs that must be declared as in [here](https://github.com/apollo-lhc/cm_mcu/blob/78ae126eeb029e2c4a4f7a6b1d28e2ab8be7bde0/projects/cm_mcu/LocalTasks.c#L275-L276). So, in order to keep the same size, I needed to create a set of fake addresses.

Reply:
Right, I get it, but I think we should figure out something better, if not in this PR, then later.

Reply:
Noted, this will be in a new PR.